image/templates/helm/stackrox-central/templates/01-central-08-configmap.yaml.htpl

```diff
@@ -12,15 +12,21 @@ metadata:
 data:
   central-config.yaml: |
     {{- ._rox.central._config | nindent 4 }}
-    {{- if ._rox.central.enableCentralDB }}
+[< if .FeatureFlags.ROX_POSTGRES_DATASTORE >]
```

Comment:
I wonder if this shouldn't have better gone into a separate config file.

Reply:
Yeah will file this as a follow up

Reply:
![image](https://user-images.githubusercontent.com/2822367/191574087-52654661-bd1d-4dab-9706-96882b45e22d.png)